src/GafferImage/ImageWriter.cpp

```diff
@@ -869,6 +935,12 @@ IECore::MurmurHash ImageWriter::hash( const Context *context ) const
 void ImageWriter::execute() const
 {
 	// Create an OIIO::ImageOutput
+	Context * c = const_cast<Context *>( Context::current() );
+	c->set( "gaffer:colorspace", defaultColorSpace()( currentFileFormat() ) );
```

Comment:
Cool - shouldn't take long for me to throw a DeleteContextVariables node together. Let's aim to get that done before we merge this.